<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertImageDialog</class>
 <widget class="QDialog" name="InsertImageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert Image</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select an image: </string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>451</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="browseButton">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>235</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="insertButton">
      <property name="text">
       <string>Insert</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="altText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>451</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>301</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Alt text (for screen readers):</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
